src/ppx_deriving_main.cppo.ml

```diff
@@ -1,101 +0,0 @@
-open Ppxlib
```

Comment:
I think clearing out this file is responsible for breaking compat with ocamlbuild!

CC @jeremiedimino @kit-ty-kate @thierry-martinez